6-appsource/cymbal-bank/modules/env_baseline/gateway.tf

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "ip_address" {
+  source  = "terraform-google-modules/address/google"
+  version = "~> 3.2"
+
+  project_id   = var.cluster_project_id
+  address_type = "EXTERNAL"
+  region       = "global"
+  global       = true
+  names        = ["mcg-ip"]
+}
+
+module "cloud-ep-dns" {
+  source  = "terraform-google-modules/endpoints-dns/google"
```

Comment:
Just noticed this module is technically deprecated - not sure we want to take a new dependency on it?

Reply:
oh, i did not know that. where do you see it's deprecated ? i see the solutions we published are still using this way to get a dns record.

Reply:
Yeah - It's still available/usable, just not sure if we want to take a new dependency on it?

Reply:
switched to use TF provider directly